frontendcomponent/androidcomponent/src/main/java/io/github/droidkaigi/confsched2019/item/HeaderItem.kt

```diff
@@ -0,0 +1,17 @@
+package io.github.droidkaigi.confsched2019.item
+
+import android.widget.TextView
+import com.xwray.groupie.Item
+import com.xwray.groupie.ViewHolder
+import io.github.droidkaigi.confsched2019.widget.component.R
+
+class HeaderItem(private val title: CharSequence) : Item<ViewHolder>() {
+
+    override fun getLayout(): Int = R.layout.section
```

Comment:
the naming `section` sounds a little bit too generic, so I was confused at first.. do you think you could change it to something more specific? Maybe something like `view_section_header`?

Reply:
Sorry 🙇 Be careful.
I will change `view_section_header`.